docs/DevOps.md

```diff
@@ -73,7 +73,7 @@ Any warnings or errors will be printed out to the console.
 
 Here are the steps to create a new release.
 
-1. Update the version number in [`MainApp.java`](https://github.com/se-edu/addressbook-level3/tree/master/src/main/java/seedu/address/MainApp.java).
+1. Update the version number in [`MainApp.java`](https://github.com/se-edu/addressbook-level3/tree/master/src/main/java/seedu/propAddress/MainApp.java).
```

Comment:
did you replace this also? I don't think this was necessary if you didn't change the actual file path
![image](https://github.com/AY2324S1-CS2103T-W11-2/tp/assets/122993267/3c1fdb22-f80e-4047-9129-0db384838825)